hadoop-hdfs-project/hadoop-hdfs-native-client/src/main/native/libhdfs/CMakeLists.txt

```diff
@@ -57,7 +57,7 @@ set(LIBHDFS_VERSION "0.0.0")
 set_target_properties(hdfs PROPERTIES
     SOVERSION ${LIBHDFS_VERSION})
 
-build_libhdfs_test(test_libhdfs_ops hdfs_static test_libhdfs_ops.c)
+build_libhdfs_test(test_libhdfs_ops hdfs_static test_libhdfs_ops.c ../libhdfs/hdfs.c)
```

Comment:
I thought `hdfs_static` includes the libhdfs/hdfs.c object file, isn't it?

Reply:
Yeah, I was testing something locally and I had to add this. I'll remove this before merging.